pg-core/src/java/org/pg/Config.java

```diff
@@ -107,6 +114,98 @@ public Builder(final String user, final String database) {
             this.pgParams.put("application_name", Const.APP_NAME);
         }
 
+
```

Comment:
My main concern is the following: in your PR, we build a Config object either from a URI line, or from a Clojure map. But I'm afraid that it's not possible to pass all the objects through the URI. Thus, maybe it's better to merge settings, namely:

1. if connection-uri is passed, parse it and get a map;
2. merge it with the rest options;
3. pass the final map into the `->config` function.

Reply:
This way, we don't need to build the `Config` object from URI, it will stay the same.

Reply:
What do you think?